plugin/settings/settings_storage.py

```diff
@@ -257,10 +256,15 @@ def __are_language_tags_valid(self):
                 error_msg = "No '{}' in syntaxes '{}'".format(
                     lang_tag, self.target_compilers)
                 return False, error_msg
-            if lang_tag not in self.compiler_args_to_get_builtins:
-                error_msg = "lang '{}' is not in {}".format(
-                    lang_tag, self.compiler_args_to_get_builtins)
-                return False, error_msg
+            elif isinstance(self.target_compilers[lang_tag],dict):
```

Comment:
I think in the current version we don't check that `target_compilers` actually has this `lang_tag` right? It seems the removed code should still be present as we need to ensure that the language tag is actually present. Then, if it is, this check should take place.